corehq/ex-submodules/soil/progress.py

```diff
@@ -141,9 +141,9 @@ def progress_complete():
     elif is_ready:
         state = STATES.success
     elif task and _is_task_pending(task):
-        state = STATES.missing
-    elif progress.percent is None:
         state = STATES.not_started
```

Comment:
would calling this `pending` make more sense than `not_started`?

Reply:
Seems likely.  @dannyroberts was there a reason ```STATES.not_started``` was picked over ```STATES.pending```? See this commit: dddeef89.  Perhaps it was to distinguish between the celery task state ```"PENDING"```?